plugins/telemetry-otel/src/main/java/org/opensearch/telemetry/tracing/exporter/SpanExporterFactory.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.telemetry.tracing.exporter;
+
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+import java.lang.invoke.MethodHandles;
+import java.lang.invoke.MethodType;
+import java.security.AccessController;
+import java.security.PrivilegedActionException;
+import java.security.PrivilegedExceptionAction;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.opensearch.SpecialPermission;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.telemetry.OtelTelemetrySettings;
+
+/**
+ * Factory class to create the {@link SpanExporter} instance.
+ */
+public class SpanExporterFactory {
```

Comment:
I believe the convention used in this plugin would need this class to be named `OTelSpanExporterFactory`